src/utility/filter-functions.js

```diff
@@ -0,0 +1,55 @@
+const filterByLabel = (allNotes, label) => {
+  const { home, food, office } = label;
+  if (food && office && home) {
+    return allNotes;
+  } else if (food && office) {
+    return allNotes.filter((note) => note.label !== "home");
+  } else if (food && home) {
+    return allNotes.filter((note) => note.label !== "office");
+  } else if (office && home) {
+    return allNotes.filter((note) => note.label !== "food");
+  } else if (office) {
+    return allNotes.filter((note) => note.label === "office");
+  } else if (home) {
+    return allNotes.filter((note) => note.label === "home");
+  } else if (food) {
+    return allNotes.filter((note) => note.label === "food");
+  } else {
+    return allNotes;
+  }
+};
```

Comment:
Switch statement can improve the readability of this block of code.

Reply:
in this case, i think both will be same